ambari/datadog_checks/ambari/data/conf.yaml.example

```diff
@@ -119,6 +119,50 @@ instances:
 #
 # logs:
 #   - type: file
-#     path: /var/log/ambari-server/*.log
```

Comment:
Question. Is using `path: /var/log/ambari-server/*.log` with wildcard doesn't work well with multiple log_processing_rules/multiline ?

I will probably have a similar case for Gitlab.

Reply:
Yes, the idea is that multiline processing rules might not apply to all the files and in that case you do have to specify each of them.

It's a pain but unless there is a pattern ( like *-error.log) we have to put the full list.

Reply:
Thx for the explanation.